pkgs/development/libraries/rdkafka/default.nix

```diff
@@ -1,13 +1,13 @@
 { lib, stdenv, fetchFromGitHub, zlib, zstd, pkg-config, python3, openssl, which, curl }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "rdkafka";
   version = "2.5.3";
 
   src = fetchFromGitHub {
     owner = "confluentinc";
     repo = "librdkafka";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

Reply:
Really, shouldn't that kind of thing be handled by some treewide automatic rewrites? Gonna cause a lot of review noise, I think.

Reply:
I fully agree with you and I can't wait this to arrive... but AFAIK, I haven't seen yet such an initiative.

I added this non-blocking comment just to inform the contributor, it was not mandatory, but I'm glad you made it.

Going to merge this PR as soon as CI is green.

Thank you!